File: 6/Problem4.c
#include<stdio.h>

void Table(int iNo)
{
    int iCnt = 0;
    int iT = 0;
    for(iCnt = 1; iCnt <= 10; iCnt++)
    {
        iT = iNo * iCnt;
        printf("%d\n",iT);
    }
    
    
}
int main()
{
    int iValue = 0;
     printf("Enter the number : ");
     scanf("%d",&iValue);

     Table(iValue);

     return 0;
}

File: 21/problem4.java
import java.util.*;
class DigitX
{
    public static void DisplayFactors(int iNo)
    {
        int iCnt = 0;
        int iSum = 0;
        for( iCnt = 1; iCnt <= iNo/2; iCnt++)
        {
            if(iNo % iCnt == 0)
            {
                iSum = iSum + iCnt;
            }
        }
        System.out.println(iSum);
    }
}


class problem4
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        DigitX.DisplayFactors(iValue);
    }
}